x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/alerting/update.ts

```diff
@@ -132,6 +132,8 @@ export default function createUpdateTests({ getService }: FtrProviderContext) {
                 created_at: response.body.created_at,
                 updated_at: response.body.updated_at,
                 execution_status: response.body.execution_status,
+                ...(response.body.next_run ? { next_run: response.body.next_run } : {}),
```

Comment:
testing these fields is a bit tricky since they may or may not exist depending on if the task runner has run these rules. I think the compromise is I can conditionally test these like you suggested if they do exist.